src/Bundle/CoverallsBundle/Collector/CiEnvVarsCollector.php

```diff
@@ -110,6 +111,48 @@ protected function fillTravisCi()
         return $this;
     }
 
+    /**
+     * Fill Github Actions environment variables.
+     *
+     * @return $this
+     */
+    protected function fillGithubActions()
+    {
+        if (!isset($this->env['GITHUB_ACTIONS'])) {
+            return $this;
+        }
+        $this->env['CI_NAME'] = 'github-actions';
```

Comment:
I changed this to `github` to comply with official action

https://github.com/coverallsapp/github-action/blob/master/src/run.ts#L24

Reply:
I revert to `github`, but sure that name is optional